vkrunner/vr-script.h

```diff
@@ -62,6 +60,12 @@ struct vr_script_shader {
         char source[];
 };
 
+struct vr_script_tolerance {
+        double value[4];
+        bool is_percent;
+        bool single_value;
```

Comment:
Instead of having `single_value` and then having to have special cases everywhere to handle it, why not just copy the single tolerance value into all of components when only one value is specified?